rules/github_rules/github_advanced_security_change.py

```diff
@@ -0,0 +1,31 @@
+from panther_base_helpers import github_alert_context
+
+# pylint: disable=line-too-long
+
+# {GitHub Action: Alert Severity}
+ADV_SEC_ACTIONS = {
```

Comment:
I would add a single comment here saying "all advanced security actions apply organization wide" and remove the per-line comments ( or otherwise restructure them to fit in line-length )

Reply:
Agreed, was trying to add more context about the actions but this is messy. I removed them.